src/main/java/com/cloudbees/jenkins/plugins/docker_build_env/DockerBuildWrapper.java

```diff
@@ -51,9 +54,13 @@ public boolean isVerbose() {
         return verbose;
     }
 
+    public boolean isPrivileged() {
```

Comment:
Javadoc would be useful. Don't forget to add <code>@since TODO</code> to let the merger specify the correct version during the merge